_layouts/default.html

```diff
@@ -66,34 +66,54 @@
             | where_exp:"item", "item.nav_exclude != true"
             | size %}
         {% if pages_top_size > 0 %}
-          {% include nav.html pages=site.html_pages key=nil %}
+          {% include nav.html pages=site.html_pages key=nil tab_index="1" expanded=true %}
```

Comment:
I dont think this is a good idea. It looks like you made this change to get around the keyboard trap caused by the search bar, but this is not a good idea since this makes navigating to the rest of the app harder.